examples/GaussExample/mainExampleGauss.m

```diff
@@ -27,7 +27,6 @@
 end
 
 opt                    = PestoSamplingOptions();
-opt.rndSeed            = 3;
```

Comment:
Is it really necessary to have always the same results in all examples? Since we are not providing any reference values, I think it doesn't really matter.